gcloud-java-examples/README.md

```diff
@@ -69,6 +69,23 @@ To run examples from your command line:
   $mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.DatastoreExample" -Dexec.args="my_name delete"
   ```
 
+  Here's an example run of `DnsExample`.
+  
+  Note that you have to enable the Google Cloud DNS API on the [Google Developers Console][developers-console] before running the following commands.
+  Note that the example creates and deletes dns records of type A only. Operations with other record types are not implemented in the example.
+  ```
+  $mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.DnsExample" -Dexec.args="create some-sample-zone elaborateexample.com. description"
+  $mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.DnsExample" -Dexec.args="get some-sample-zone"
```

Comment:
Is listing eventually consistent?  If so, does the output usually show the created zone by the time the next command is entered, or does it usually take longer to show up?  It might be worth adding an additional note before the example explaining that the user may have to wait before the newly created zone appears.